src/test/pmem_memmove/pmem_memmove.c

```diff
@@ -92,6 +92,27 @@ swap_mappings(char **dest, char **src, size_t size, int fd)
 	*src = ts;
 }
 
+/*
+ * verify_contents -- verify that buffers match, if they don't - print contents
+ * of both and abort the test
+ */
+static void
+verify_contents(const char *file_name, int test,
+		const char *buf1, const char *buf2,
+		size_t len)
+{
+	int r = memcmp(buf1, buf2, len);
+	if (r == 0)
```

Comment:
r variable is used only for memcmp. IMHO Compare operation could be used directly in if condition without using additional variable

Reply:
Done.